.circleci/config.yml

```diff
@@ -7,8 +7,9 @@ version: 2.1
 defaults: &defaults
   docker:
     - image: cimg/android:2021.12.2
+  resource_class: large
   environment:
-    JVM_OPTS: -Xmx3200m
+    JVM_OPTS: -Xmx3600m
```

Comment:
Are these changes related?

Reply:
No it is a problem related to CI and failing build. I extracted it in a separated PR